### ISSUE TITLE
renderer.setClearColorHex() is is not a function

### ISSUE THREAD

**Issue by bengben** [josdirksen/learning-threejs#13]
In:【 learning-threejs/chapter-01/02-first-scene.html 】

the method setClearColorHex()  has changed to setClearColor()  ?

**Comment by feeltheburn** [comment]
Correct... the book gets odd in that area, as later snippets in the book show 

(page 16)
renderer.setClearColorHex()
renderer.setClearColor(new THREE.Color(0xEEEEEE));

which is also hosed.

The correct line is:

renderer.setClearColor(new THREE.Color(0xEEEEEE, 1.0));

**Comment by dehuszar** [comment]
renderer.setClearColor(0xEEEEEE, 1) also works.  I didn't need to use THREE.Color at all.
